spring-boot-project/spring-boot-tools/spring-boot-configuration-processor/src/json-shade/java/org/springframework/boot/configurationprocessor/json/JSONObject.java

```diff
@@ -596,12 +596,10 @@ public String optString(String name, String fallback) {
 	 */
 	public JSONArray getJSONArray(String name) throws JSONException {
 		Object object = get(name);
-		if (object instanceof JSONArray) {
-			return (JSONArray) object;
-		}
-		else {
+		if (!(object instanceof JSONArray)) {
```

Comment:
This is a shaded version of a json library so we shouldn't modify it. Can you please revert all changes in this file?